### DISCUSSION TITLE
cn/2024/12/17/others-conclusions/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#199] [Announcements]
# cn/2024/12/17/others-conclusions/

https://hongtaoh.com/cn/2024/12/17/others-conclusions/

**Comment by earfanfan** [comment]
>我觉得人活在世间，就是要经过不断思考、不断体验、不多认识自己和认识世界，让自己越来越接近真理，越来越接近无所不知无所不晓的上帝。

啊，这，前半段我是认同的，因为活在人群中总是会不断接收自我以外各种渠道的信息，坐个电梯都会无意识被电梯广告袭击大脑，很多东西确实需要自己去思考辨析。至于后半段，既然你觉得命理有时对你更有用，那你相信人类会被命运主宰吗？或者说，世界上真得存在绝对的真理嘛？再或者说，难道正是因为你相信不断认识自己和世界就能越来越接近真理和上帝，你才愿意这么做的么。

对了，你那里现在应该也是冬天吧，每天是什么天气，温度几何，外面的景色是树都秃得差不多了，只剩几片树叶还坚挺着不肯落下，还是秋冬之交，梧桐树才刚刚变得黄橙橙一片咧。

**Reply by hongtaoh** [reply]
>既然你觉得命理有时对你更有用，那你相信人类会被命运主宰吗？或者说，世界上真得存在绝对的真理嘛？再或者说，难道正是因为你相信不断认识自己和世界就能越来越接近真理和上帝，你才愿意这么做的么。

我目前的认知：每个人都有自己的命运；这个命运是灵魂进入地球之前和一群指导灵商量好定下来的，其中前 20 年的命运很少会有改动的机会，但后面的会有自己调整的空间；每个人的命运之所以存在是为了让灵魂进化。

真理这个词太大了，囊括的东西太多。但我相信是有最终的真理的，也就是我的所有疑问都可以得到答案。至于是否不断认识自己和认识世界就能接近真理，这个我也不知道。但对自己和世界认识加深，至少可以让自己减少大部分的烦恼，这个好处足够了。

我最近这里下雪了。最冷的时候大概零下二十度，景观和温度和东北差不多，比北京要冷。

**Reply by earfanfan** [reply]
本来嘛，本南方人听到下雪应该会激动地跑出去玩耍一首歌的时间的，不过今早遛狗的时候下雪子了，感受到了一种寒气钻进衣服里的冷。

其实从你的认知似乎可以反推你的人生经历，不过我就不推理了。从一种角度来说，出身寒门，当今大约只能通过狭窄的教育之路来获取改变的机会，而我们犹未可知，那些投胎投到富裕或权贵人家的，从小过的是什么样的生活，到底拥有比我们多多少的选择。虽然我和你的认知的角度不同，不过现阶段的结论好像是相似的，我想等大家走到人生终点的时候，结论仍然是不同的吧。

**Reply by hongtaoh** [reply]
推理大师，欢迎提交你的推理申请。

>出身寒门，当今大约只能通过狭窄的教育之路来获取改变的机会

不啊，如果是女生，可以做主播。

出身寒门怎么做才能有改变，这是一个很大的话题，但应该肯定不止教育这一条路。而且目前很明显的是这条路能带来的改变越来越有限。它更多的是一张门票，剩下的还是要靠自己努力以及凭运气。

**Reply by earfanfan** [reply]
>出身寒门，当今大约只能通过狭窄的教育之路来获取改变的机会

额，看来我也是通过我的经验知识来得到的结论，也很有限的说。

>推理大师，欢迎提交踢的推理申请。

TO hongtaoh：俺提交申请，请予通过。（PS如果你通过的话，下个月你生日的时候我可以送两个生日蛋糕给你的说）。

>它更多的是一张门票，剩下的还是要靠自己努力以及凭运气。

那是当然，现在也并不是读了大学就一定能找到工作、过上安稳生活了。

**Reply by hongtaoh** [reply]
欢迎淘宝海运生日蛋糕过来。提前谢谢袁姐。

**Reply by earfanfan** [reply]
>欢迎淘宝海运生日蛋糕过来。

这令我想起了[一台饺子皮机走丝绸之路](https://www.douban.com/note/849051180/?_i=4933945I6dZ-EB)，蛋糕在漂洋过海的路途中会坏掉的。其实是我最近在学画画，上周末突然想到也许等你下个月生日的时候画两个生日蛋糕送给你，小红书上搜新手入门画还真的有画蛋糕的，理论上来讲，等到下个月的时候我画的应该勉强能入眼了，现在画的丑得让我崩溃。

又及，理论上来说，将来说不定我就学会做蛋糕了，等猴年马月你到武汉溜达的话，也许我可以撸起袖子做两个蛋糕给你吃。

>提前谢谢袁姐。

你不要称我姐，像小闫同学那种小朋友称我姐才对。而且我喜欢的人喜欢甜妹，我最近要当甜妹，哈哈哈。

**Reply by hongtaoh** [reply]
>而且我喜欢的人喜欢甜妹，我最近要当甜妹，哈哈哈。

恭喜袁姐。

>其实是我最近在学画画

那就静等袁姐的画了。

**Reply by earfanfan** [reply]
>恭喜袁姐。

人类转瞬即逝的喜欢并没什么好喜的，若将来有值得欢喜的时候我再通知你恭喜我吧。不过我都这么说了，你居然还要喊我姐。唉，我错了，你这语言风格都被客气的抱歉、谢谢、恭喜包裹着，让我忘记你本来很倔强了。我以为我是不撞南墙不回头、撞了南墙撞倒了南墙依旧不回头，现在看来你好像是不管撞倒多少南墙都不回头的那种。哈哈，我好想喊益辉来看，还有比我都倔好多的人呢，说不定你俩会因为各自的倔强而惺惺相惜。

**Reply by hongtaoh** [reply]
>人类转瞬即逝的喜欢并没什么好喜的，若将来有值得欢喜的时候我再通知你恭喜我吧。

好的袁姐。静候佳音。
